main.go

```diff
@@ -68,7 +68,7 @@ func main() {
 	api.HandleFunc("/verify/{key}", h.VerifyKey).Methods(http.MethodGet, http.MethodOptions)
 
 	books := api.PathPrefix("/book").Subrouter()
-	books.Use(m.Authenticated)
+	books.Use(m.WithUserModel)
```

Comment:
Does `WithUserModel` imply `Authenticated`? Otherwise we'd need both.

Reply:
Seems to work?